archivers/par2/Portfile

```diff
@@ -17,20 +18,18 @@ license           GPL-2
 maintainers       jabber.org:julian
 categories        archivers
 platforms         darwin 
-homepage          http://parchive.sourceforge.net/
-master_sites      sourceforge:parchive
+homepage          https://github.com/Parchive/par2cmdline
 
-checksums         md5 1551b63e57e3c232254dc62073b723a9 \
-                  sha1 2fcdc932b5d7b4b1c68c4a4ca855ca913d464d2f \
-                  rmd160 e98dc84fa7a4304b8d71c1d373fdce0bca1e6290
+checksums         sha1 96713067eb29ef18aba64c5c168c64d6f72c8300 \
```

Comment:
`sha1` checksum can be dropped, we don't use that anymore

Reply:
Done.